schemas/tlc/1.1/sxl.yaml

```diff
@@ -886,6 +886,7 @@ objects:
               
               Note:
               - The checksum should be calculated using the same data as used in S0098
+              - Downloaded data with S0098, hashed with SHA-2, should match this value.
```

Comment:
suggestion:
Data downloaded with S0098 and hashed with SHA-2 should match this value.

Reply:
Updated text